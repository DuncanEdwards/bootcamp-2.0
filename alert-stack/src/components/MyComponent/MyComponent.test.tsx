/* eslint-disable testing-library/no-debugging-utils */
import { render, screen } from "@testing-library/react";
import { MyComponent } from "./MyComponent";

test("Render a basic my component", () => {
  const { container } = render(
    <MyComponent
      isEmployed={true}
      location="Bristol"
      name={{ firstName: "Duncan", lastName: "Edwards" }}
    />
  );

  expect(container).toMatchSnapshot();

  //   expect(screen.getByText("Duncan Edwards")).toBeInTheDocument();
  //   expect(screen.getByText("Location: Bristol")).toBeInTheDocument();
  //   expect(screen.getByText("IsEmployed: Yes!")).toBeInTheDocument();
});
