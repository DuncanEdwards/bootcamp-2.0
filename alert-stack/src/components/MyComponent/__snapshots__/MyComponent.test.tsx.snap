// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a basic my component 1`] = `
<div>
  <div>
    Edwards, Duncan
  </div>
  <div>
    Location: 
    Bristol
  </div>
  <div>
    IsEmployed?: 
    Yes!
  </div>
</div>
`;
